chirp/drivers/th9000.py

```diff
@@ -807,6 +867,96 @@ def _filter(name):
                               int(_freqrange.rxrangehi) / 10))
         freqrange.append(rs)
 
+        rs = RadioSetting(
+            "btn_p1", "P1 Short Press",
+            RadioSettingValueList(
```

Comment:
There's a newer/better way to do this, which is `RadioSettingValueMap` which tolerates  invalid data better than the `List` variant. This is okay as it is, but you might want to consider converting these to use the map method for better safety in the future.

Reply:
You did use `current_index` though, which is also good for safety. Just FYI is all.